WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -139,14 +143,30 @@ + (void)setShouldWaitForQuiescence:(BOOL)value
 
 #pragma mark Private
 
-+ (NSRange)bindingPortRangeFromArguments
++ (NSString*)valueFromArguments: (NSArray<NSString *> *)arguments forKey: (NSString*)key
 {
-  NSArray *arguments = NSProcessInfo.processInfo.arguments;
-  NSUInteger index = [arguments indexOfObject:@"--port"];
+  NSUInteger index = [arguments indexOfObject:key];
   if (index == NSNotFound || index == arguments.count - 1) {
-    return NSMakeRange(NSNotFound, 0);
+    return nil;
   }
-  NSString *portNumberString = arguments[index + 1];
+  return arguments[index + 1];
+}
+
++ (NSUInteger)mjpegServerPortFromArguments
+{
+  NSString *portNumberString = [self valueFromArguments: NSProcessInfo.processInfo.arguments
+                                                 forKey: @"--mjpeg-server-port"];
+  NSUInteger port = (NSUInteger)[portNumberString integerValue];
```

Comment:
here we should check if portNumberString is not nil

Reply:
When `portNumberString==nil` calling `[portNumberString integerValue]` will return `0`. Which is what we expect, as previous both cases when the port value is not specified or set to zero will fallback return `NSNotFound` and fall back to using env or default value upstream.